### DISCUSSION TITLE
Webdriver v6 uses Chrome incompatible with new Macbook M2 chip

### DISCUSSION THREAD

**Discussion by alisman** [webdriverio/webdriverio#12193] [Q&A]
I have recently upgrade to Macbook with M2 chip.  WDIO (v6) boots up a version of Chrome which would appear to be incompatible with M2.  The browser opens but I think it's using some virtualization layer which results in extremely sluggish performance.  Is there a way to force WDIO to use the "normal" Chome executable I use for everyday work?

**Comment by christian-bromann** [comment]
> WDIO (v6)

This version has been deprecated a long time ago. We strongly recommend to update your WDIO version.

**Reply by alisman** [reply]
I get it but I can't rewrite my 700+ tests for async right now.  There is no way to get it to use a different browser?  Maybe just install a different ChromeDriver?

**Reply by christian-bromann** [reply]
We have a [codemod](https://webdriver.io/docs/async-migration) that can help with the migration. You won't be able to run tests on this version forever unfortunately.

I have no idea why Chrome appears sluggish, I doubt it has to do with WebdriverIO but it is hard to tell. You can automate any other browser if you want.
